test/cmd/images.sh

```diff
@@ -20,125 +21,125 @@ USE_IMAGES=${USE_IMAGES:-$defaultimage}
 
 # This test validates images and image streams along with the tag and import-image commands
 
-oc get images
-oc create -f test/integration/fixtures/test-image.json
-oc delete images test
+os::cmd::expect_success 'oc get images'
+os::cmd::expect_success 'oc create -f test/integration/fixtures/test-image.json'
+os::cmd::expect_success 'oc delete images test'
 echo "images: ok"
 
-oc get imageStreams
-oc create -f test/integration/fixtures/test-image-stream.json
+os::cmd::expect_success 'oc get imageStreams'
+os::cmd::expect_success 'oc create -f test/integration/fixtures/test-image-stream.json'
 # verify that creating a registry fills out .status.dockerImageRepository
-if [ -z "$(oc get imageStreams test --template="{{.status.dockerImageRepository}}")" ]; then
+if os::cmd::expect_success_and_not_text "oc get imageStreams test --template='{{.status.dockerImageRepository}}'" '.'; then
   # create the registry
-  oadm registry --credentials="${KUBECONFIG}" --images="${USE_IMAGES}" -n default
+  os::cmd::expect_success "oadm registry --credentials=${KUBECONFIG} --images='${USE_IMAGES}' -n default"
   # make sure stream.status.dockerImageRepository IS set
-  [ -n "$(oc get imageStreams test --template="{{.status.dockerImageRepository}}")" ]
+  os::cmd::expect_success_and_text "oc get imageStreams test --template='{{.status.dockerImageRepository}}'" 'test'
 fi
-oc delete imageStreams test
-[ -z "$(oc get imageStreams test --template="{{.status.dockerImageRepository}}")" ]
-
-oc create -f examples/image-streams/image-streams-centos7.json
-[ -n "$(oc get imageStreams ruby --template="{{.status.dockerImageRepository}}")" ]
-[ -n "$(oc get imageStreams nodejs --template="{{.status.dockerImageRepository}}")" ]
-[ -n "$(oc get imageStreams wildfly --template="{{.status.dockerImageRepository}}")" ]
-[ -n "$(oc get imageStreams mysql --template="{{.status.dockerImageRepository}}")" ]
-[ -n "$(oc get imageStreams postgresql --template="{{.status.dockerImageRepository}}")" ]
-[ -n "$(oc get imageStreams mongodb --template="{{.status.dockerImageRepository}}")" ]
+os::cmd::expect_success 'oc delete imageStreams test'
+os::cmd::expect_failure 'oc get imageStreams test'
+
+os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
+os::cmd::expect_success_and_text "oc get imageStreams ruby --template='{{.status.dockerImageRepository}}'" 'ruby'
```

Comment:
Tightened all `grep`s here from anything to the name of the imagestream being fetched